README.md

```diff
@@ -45,20 +58,48 @@ Password: Dynamic33
 
 5. **Run the Analysis Code**
 
-In RStudio, open `vancouver-tree-height-geo.qmd` and run all code chunks.
+open a terminal (from the docker rstudio) and run the following commands:
+
+```
```

Comment:
the first line of backticks should have the identifier `bash` so that github knows it needs to use bash code highlighting. like this:

\```bash

This should be applied to all chunks. See examples further up in the file